### PR TITLE
Feature/columns

### PR DESCRIPTION
**Pull request by iruzevic**
* fixing storybook
* fixing example field
* adding hasWrapper, hasSimpleWrapper disableWrapper
* updating packages
* adding options on column items to have spacing top bottom

For some reason storybook is not working on sizes larger than medium. Some bug in the latest package versions :(

### CODE REVIEW

File: blocks/init/src/blocks/custom/quote/manifest.json [hunk 1]
```diff
@@ -12,8 +12,10 @@
     "Quote"
   ],
   "example": {
-    "author": "Narcotics Anonymous",
-    "content": "Insanity is doing the same thing, over and over again, but expecting different results."
+    "attributes": {
+      "author": "Narcotics Anonymous",
```
Comment (dingo-d) [inline blocks/init/src/blocks/custom/quote/manifest.json:16]: Narcotics Anonymous? 😬 

That's... pretty bleak 👀

File: blocks/init/src/blocks/custom/video/manifest.json [hunk 2]
```diff
@@ -12,9 +12,11 @@
     "preview"
   ],
   "example": {
-    "media": {
-      "id": 0,
-      "url": "https://storage.googleapis.com/coverr-main/mp4%2FIn-The-Clouds.mp4"
+    "attributes": {
+      "media": {
+        "id": 0,
+        "url": "https://storage.googleapis.com/coverr-main/mp4%2FIn-The-Clouds.mp4"
```
Comment (dingo-d) [inline blocks/init/src/blocks/custom/video/manifest.json:18]: Whose video is this, and what is the license of it? Can we use it in our project?

File: blocks/init/src/blocks/wrapper/components/wrapper-responsive-tab-content-simple.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    onChangeHideBlock,
+  } = props;
+
+  return (
+    <Fragment>
+      {onChangeSpacingTop &&
+        <RangeControl
+          label={
+            <Fragment>
+              <Icon icon={() => icons.spacingTop} />
```
Comment (dingo-d) [inline blocks/init/src/blocks/wrapper/components/wrapper-responsive-tab-content-simple.js:31]: Why do we need to pass the icon name as an arrow function? Can this be extracted above the `return`?

File: blocks/init/src/blocks/wrapper/wrapper.php [hunk 4]
```diff
@@ -10,36 +10,54 @@
...
-  $wrapper_class = Components::classnames([
-    $wrapper_main_class,
-    isset( $attributes['styleBackgroundColor'] ) && ! empty( $attributes['styleBackgroundColor'] ) ? "{$wrapper_main_class}__bg-color--{$attributes['styleBackgroundColor']}" : '',
-    $attributes['styleSpacingTop'] ? Components::responsive_selectors($attributes['styleSpacingTop'], 'spacing-top', $wrapper_main_class) : '',
-    $attributes['styleSpacingBottom'] ? Components::responsive_selectors($attributes['styleSpacingBottom'], 'spacing-bottom', $wrapper_main_class) : '',
-    $attributes['styleHideBlock'] ? Components::responsive_selectors($attributes['styleHideBlock'], 'hide-block', $wrapper_main_class, true) : '',
-  ]);
+$wrapper_class = Components::classnames([
+  $wrapper_main_class,
+  isset( $attributes['styleBackgroundColor'] ) && ! empty( $attributes['styleBackgroundColor'] ) ? "{$wrapper_main_class}__bg-color--{$attributes['styleBackgroundColor']}" : '',
```
Comment (dingo-d) [inline blocks/init/src/blocks/wrapper/wrapper.php:32]: Why `isset()` and `! empty()` check? Usually `! empty` will suffice.

File: blocks/init/src/blocks/custom/quote/manifest.json (hunk 1 above)
Comment (iruzevic) [reply blocks/init/src/blocks/custom/quote/manifest.json]: fixed

File: blocks/init/src/blocks/custom/video/manifest.json (hunk 2 above)
Comment (iruzevic) [reply blocks/init/src/blocks/custom/video/manifest.json:18]: I have used a placeholder video from this list

https://gist.github.com/jsturgis/3b19447b304616f18657

File: blocks/init/src/blocks/wrapper/components/wrapper-responsive-tab-content-simple.js (hunk 3 above)
Comment (iruzevic) [reply blocks/init/src/blocks/wrapper/components/wrapper-responsive-tab-content-simple.js]: fixed, it was left from the previous implementation

File: blocks/init/src/blocks/wrapper/wrapper.php (hunk 4 above)
Comment (iruzevic) [reply blocks/init/src/blocks/wrapper/wrapper.php]: fixed

**Comment by iruzevic** [comment]
I have also renamed gutter and horizontal spacing

File: blocks/init/src/blocks/custom/video/manifest.json (hunk 2 above)
Comment (dingo-d) [reply blocks/init/src/blocks/custom/video/manifest.json:18]: Couldn't find it on that list. We should check the license and add attribution if reuse is permitted.

Comment (iruzevic) [reply blocks/init/src/blocks/custom/video/manifest.json:18]: I will open an issue

File: blocks/init/src/blocks/custom/image/transforms.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import { createBlock } from '@wordpress/blocks';
+import manifest from './../../manifest.json';
+import manifestParagraph from './manifest.json';
+import manifestCarouselImage from './../carousel-image/manifest.json';
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/blocks/custom/image/transforms.js:4]: yea good catch. fixed

File: blocks/init/src/blocks/custom/carousel-image/transforms.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+import manifestImage from './../image/manifest.json';
+
+export const Transforms = {
+  from: [
+    {
+      type: 'block',
+      blocks: [`${manifest.namespace}/${manifestImage.blockName}`],
+      transform: ({ media }) => (
+        createBlock(`${manifest.namespace}/${manifestParagraph.blockName}`, {
+          media,
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply blocks/init/src/blocks/custom/carousel-image/transforms.js:13]: yea good catch. fixed

**Pull request merged**